graspy/embed/lse.py

```diff
@@ -58,10 +58,11 @@ class LaplacianSpectralEmbed(BaseEmbed):
 
     """
 
-    def __init__(self, method=selectSVD, *args, **kwargs):
+    def __init__(self, form='DAD', method=selectSVD, *args, **kwargs):
```

Comment:
Form default was I-DAD before and now its DAD?

Reply:
correct!